tests/test_cli_integration.py

```
@@ -0,0 +1,27 @@
+import sys
+from pathlib import Path
+
+def test_cli_analyze_gpx():
+    """Verify that biosystems.cli analyze works on a real GPX file."""
+    # Use one of the real GPX files in data/raw
+    gpx_file = Path("data/raw/20250503_174110_baseox_wk1_sat_day6_proglong_30_40min_145_160bpm_11-14minmi_Z2_durability_rpe_4_anklepain_2.gpx")
+    
+    if not gpx_file.exists():
+        pytest.skip(f"Test file {gpx_file} not found")
+        
+    result = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)